### PR TITLE
fix nfd automation to be able to run on 4.12

### PR DESCRIPTION
**Pull request by pruan-rht**
4.10 https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/5835/console
4.12 https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/Runner-v3-smoke/5837/console

@lenahorsley @liqcui @wabouhamad PTAL

### CODE REVIEW

File: features/step_definitions/nfd.rb [hunk 1]
```diff
@@ -84,7 +83,14 @@
   step %Q/all the pods in the project reach a successful state/
   step %Q/I store all worker nodes to the :worker_nodes clipboard/
   step %Q/I store the masters in the :master_nodes clipboard/
-  step %Q/<%= cb.master_nodes.count %> pods become ready with labels:/, table(%{
+
+  # for OCP >= 4.12, the number of master nodes with the label 'nfd-master' is
+  # 1
+  cb.expected_master_nodes_with_label = cb.master_nodes.count
+  if env.version_gt("4.11", user: user)
```
Comment (liangxia) [inline features/step_definitions/nfd.rb:90]: Perhaps use `env.version_ge("4.12", user: user)` ?

Comment (pruan-rht) [reply features/step_definitions/nfd.rb]: @liangxia sure, it's fixed now

**Comment by pruan-rht** [comment]
PR depends on https://github.com/openshift/cucushift/pull/9488

**Pull request merged**